crates/typst/src/symbols/sym.rs

```diff
@@ -495,6 +494,59 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
         spade.stroked: '♤',
     ],
 
+    // Music.
+    note: [
+        up: '🎜',
+        down: '🎝',
+        whole: '𝅝',
+        half: '𝅗𝅥',
+        quarter: '𝅘𝅥',
+        quarter.alt: '♩',
+        eighth: '𝅘𝅥𝅮',
+        eighth.alt: '♪',
+        eighth.beamed: '♫',
+        sixteenth: '𝅘𝅥𝅯',
+        sixteenth.beamed: '♬',
+        grace: '𝆕',
+        grace.slash: '𝆔',
+        kievan.final: '𝇠',
+        kievan.whole: '𝇢',
+        kievan.half: '𝇣',
+        kievan.quarter.t: '𝇥',
+        kievan.quarter.b: '𝇤',
+        kievan.eighth.t: '𝇧',
+        kievan.eighth.b: '𝇦',
```

Comment:
I can see the ones above, but `kievan` seems very obscure. I don't think we need everything.

Reply:
I will remove them later.